### PR TITLE
add response statuses for each order

### PR DESCRIPTION
**Pull request by rocketmannot**

### CODE REVIEW

File: src/main/kotlin/com/lykke/matching/engine/services/MultiLimitOrderService.kt [hunk 1]
```diff
@@ -151,23 +154,32 @@ class MultiLimitOrderService(private val limitOrderService: GenericLimitOrderSer
                 Pair(assetPair.quotingAssetId, balancesHolder.getAvailableBalance(clientId, assetPair.quotingAssetId)))
 
         orders.forEach { order ->
+            val orderStatusBuilder = ProtocolMessages.MultiLimitOrderResponse.OrderStatus.newBuilder().setId(order.externalId)
+                    .setMatchingEngineId(order.id).setStatus(MessageStatus.OK.type).setVolume(order.volume).setPrice(order.price)
+
             if (order.price <= 0) {
+                orderStatusBuilder.status = MessageStatus.INVALID_PRICE.type
```
Comment (dmitrykrasovskih) [inline src/main/kotlin/com/lykke/matching/engine/services/MultiLimitOrderService.kt]: 1. Update original order.status (check SingleLimitOrderService)
2. Add order to trustedClientLimitOrdersReport

File: src/main/kotlin/com/lykke/matching/engine/services/MultiLimitOrderService.kt [hunk 2]
```diff
@@ -151,23 +154,32 @@ class MultiLimitOrderService(private val limitOrderService: GenericLimitOrderSer
...
                 LOGGER.info("[${order.assetPairId}] Unable to add order ${order.volume} @ ${order.price} due to invalid price")
             } else if (!order.checkVolume(assetPair)) {
+                orderStatusBuilder.status = MessageStatus.TOO_SMALL_VOLUME.type
```
Comment (dmitrykrasovskih) [inline src/main/kotlin/com/lykke/matching/engine/services/MultiLimitOrderService.kt]: 1. Update original order.status (check SingleLimitOrderService)
2. Add order to trustedClientLimitOrdersReport

File: src/main/kotlin/com/lykke/matching/engine/services/MultiLimitOrderService.kt [hunk 3]
```diff
@@ -151,23 +154,32 @@ class MultiLimitOrderService(private val limitOrderService: GenericLimitOrderSer
...
                 LOGGER.info("[${order.assetPairId}] Unable to add order ${order.volume} @ ${order.price} due too small volume")
             } else if (orderBook.leadToNegativeSpreadForClient(order)) {
+                orderStatusBuilder.status = MessageStatus.LEAD_TO_NEGATIVE_SPREAD.type
```
Comment (dmitrykrasovskih) [inline src/main/kotlin/com/lykke/matching/engine/services/MultiLimitOrderService.kt]: 1. Update original order.status (check SingleLimitOrderService)
2. Add order to trustedClientLimitOrdersReport

**Comment by rocketmannot** [comment]
pull request was closed as duplicated

**Pull request closed without merging**